### PR TITLE
Modify Demodata

### LINKED ISSUE #4: As a New User, I can view demo data
so that I can play around with the features.

### LINKED ISSUE #9: As a new user, I can clear the Demo data
... so that I can clear the data and start using the Hustlebook with my own data.

### LINKED ISSUE #62: File location for storage is wrong
Change from addressbook.json to hustlebook.json

### PR DESCRIPTION
**Pull request by kuanjr**
Demo data does not include new attributes like PrevMetDate and Info.

Let's,

- add the required attributes to demo data.
- change userguide to reflect the clear command
- change preferred filename of data from `addressbook.json` to `hustlebook.json`
- modify the way the persons object is presented in the GUI

Closes #4, closes #9, closes #62

### CODE REVIEW

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 1]
```diff
@@ -6,12 +6,7 @@
 
 import seedu.address.model.AddressBook;
 import seedu.address.model.ReadOnlyAddressBook;
-import seedu.address.model.person.Address;
-import seedu.address.model.person.Email;
-import seedu.address.model.person.Flag;
-import seedu.address.model.person.Name;
-import seedu.address.model.person.Person;
-import seedu.address.model.person.Phone;
+import seedu.address.model.person.*;
```
Comment (Vishwanth2210) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java:9]: It would be better to not optimise the imports.

**Review by Vishwanth2210 — CHANGES_REQUESTED** [review]
Looks good. Other than the imports optimisation, the rest is good for merging.

**Review by Vishwanth2210 — APPROVED** [review]
Looks good!

File: src/main/java/seedu/address/model/util/SampleDataUtil.java (hunk 1 above)
Comment (kuanjr) [reply src/main/java/seedu/address/model/util/SampleDataUtil.java]: Ok fixed

**Pull request merged**